### DISCUSSION TITLE
How to change a language of the Calendar

### DISCUSSION THREAD

**Discussion by Werionae** [kudrykv/latex-yearly-planner#115] [Q&A]
Hello, I would like to change an original language of the Calendar to my native languge (English→Czech). What should be the procedure? I need to add or change something to the command line or change something in “latex-yearly-planner” folder? Thank you

**Comment by jerry-74** [comment]
before running the ./single.sh script, set the TRANSLATION variable to the value "czech" (or call the translate.py script manually before creating PDF via xelatex). I have tried it, however, there are several issues:
- czech diacritic
- the translation is carried out after generating the text files by simply searching and replacing the English words with Czech ones; so the script cannot distinguish for example whether it translates the abbreviated name of the month or the full word ("may" --> "kvě" / "may" - "květen"); also, script does not translate short month names at all

**Reply by Werionae** [reply]
Thank you for your post, instructions and sharing your experience with the result. I've gotten used to English over the months, so I'll keep it that way, especially if the localization doesn't work correctly.
